lib/Frontend/CompilerInvocation.cpp

```diff
@@ -339,6 +339,17 @@ static bool ParseLangArgs(LangOptions &Opts, ArgList &Args,
 
     Opts.DebugConstraintSolverAttempt = attempt;
   }
+
+  for (const Arg *A : Args.filtered(OPT_debug_constraints_on_line)) {
+    unsigned line;
+    if (StringRef(A->getValue()).getAsInteger(10, line)) {
+      Diags.diagnose(SourceLoc(), diag::error_invalid_arg_value,
+                     A->getAsString(Args), A->getValue());
+      return true;
```

Comment:
If there are gonna be multiple of these, would it make sense to return after the loop if it failed, instead of bailing on the first one?

Reply:
This is actually an issue in the wider function, but I can correct that.